utils/scripts/util.py

```diff
@@ -141,3 +141,8 @@ def get_provider_account_name(string_suffix: string = "DBAAS", invalid=None):
 
 def get_project_name(string_name: list):
     return str(time.time())[-4:] + "-" + ("-".join(string_name)).lower()
+
+
+def get_instance_name(view: string):
+    # Field `name` should be a slug between 5 and 50 characters in length, containing only unicode characters, unicode numbers,hyphens, spaces, or underscores, and starting with a character.
```

Comment:
I hope some isv have restrictions on instance name length like cockroach. We can get clarified and update this later.